src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,82 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DOUBLER = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table = new Node[DEFAULT_CAPACITY];
+    private int capacity = (int) (table.length * LOAD_FACTOR);
+    private int size;
+
+    public static class Node<K, V> {
+        private K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size + 1 >= capacity) {
```

Comment:
У тебя массив будет увеличиваться при size = 11, а не 12. 
```suggestion
        if (size >= capacity) {
```